services/galley/test/integration/API/MLS/Util.hs

```diff
@@ -94,10 +94,10 @@ mapRemoteKeyPackageRef ::
   (Request -> Request) ->
   KeyPackageBundle ->
   m ()
-mapRemoteKeyPackageRef brig bundle =
+mapRemoteKeyPackageRef galleyCall bundle =
```

Comment:
Is this in fact galleyCall and it was a mistake before or is this a mistake now?